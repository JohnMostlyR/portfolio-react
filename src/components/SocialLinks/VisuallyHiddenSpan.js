import styled from 'styled-components';

import {visuallyHidden} from '../../styles/mixins';

const VisuallyHiddenSpan = styled.span`
  ${visuallyHidden}
`;

export default VisuallyHiddenSpan;