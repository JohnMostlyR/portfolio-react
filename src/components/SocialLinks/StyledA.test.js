import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import VisuallyHiddenSpan from './VisuallyHiddenSpan'

describe('VisuallyHiddenSpan', () => {
  it('Should render and match the snapshot', () => {
    const tree = renderer.create(<VisuallyHiddenSpan/>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});