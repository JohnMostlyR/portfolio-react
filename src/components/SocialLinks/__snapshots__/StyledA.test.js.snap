// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VisuallyHiddenSpan Should render and match the snapshot 1`] = `
.c0 {
  position: absolute !important;
  width: 1px !important;
  height: 1px !important;
  margin: -1px !important;
  padding: 0 !important;
  border: 0 !important;
  overflow: hidden !important;
  -webkit-clip: rect(0 0 0 0) !important;
  clip: rect(0 0 0 0) !important;
}

<span
  className="c0"
/>
`;
